pkg/eks/eks.go

```diff
@@ -195,7 +200,6 @@ func (c *ClusterProvider) CanOperate(spec *api.ClusterConfig) (bool, error) {
 	case awseks.ClusterStatusCreating, awseks.ClusterStatusDeleting, awseks.ClusterStatusFailed:
 		return false, fmt.Errorf("cannot perform Kubernetes API operations on cluster %q in %q region due to status %q", spec.Metadata.Name, spec.Metadata.Region, status)
 	default:
-		// all other states are considered operable, including UPDGRADING (which is missing from the SDK)
```

Comment:
The part about the `UPGRADING` state missing in the SDK was useful in this comment.

Reply:
Is it still missing? I see an Updating state

Reply:
https://github.com/aws/aws-sdk-go/blob/d85f343adb9b56f552867c2fa558d60f43a02242/service/eks/api.go#L6653-L6654

Reply:
It was about the `UPGRADING` state. I think the original comment might've actually meant the `UPDATING` state, as when a cluster is being upgraded, the status is `UPDATING`. So I think we're good to remove this comment.